src/js/cookies-functions.js

```diff
@@ -149,6 +148,7 @@ export function setCookie(name, value, options) {
         if (document.location.protocol === 'https:') {
             cookieString = cookieString + '; Secure';
         }
+        cookieString = cookieString + '; Secure';
```

Comment:
Was this left in by mistake? It now looks like we are doing this inside and outside of the if statement